### DISCUSSION TITLE
npx @chakra-ui/cli typegen misbehaves if it is not properly installed

### DISCUSSION THREAD

**Discussion by RobertRad** [chakra-ui/chakra-ui#9090] [General] [answered]
Hi,

I had issues with the typegen.
The cli told me, everything is fine and types were generated, but my own `visual` wasn't there.

In the end I found out that I had to install the cli into the package.json like this: `npm install -D @chakra-ui/cli`.

I seem to have it installed locally once (?), but was absolutely not aware of this. And I got no pointer in that direction.

Helpful would be printing the path to the generated typings would be helpful unconditionally.
Or at least some hint in the documentation. There only `npx ...` is mentioned.

Btw: Probably unrelated, but want to note it. I use volta to install node / npm.

**Comment by segunadebayo** [comment, marked as answer]
This is a known issue, we're looking into it.
